client/my-sites/domains/domain-search/domain-search.jsx

```diff
@@ -42,10 +43,14 @@ class DomainSearch extends Component {
 
 	state = {
 		domainRegistrationAvailable: true,
+		domainRegistrationMaintenanceEndTime: null,
 	};
 
-	handleDomainsAvailabilityChange = isAvailable => {
-		this.setState( { domainRegistrationAvailable: isAvailable } );
+	handleDomainsAvailabilityChange = ( isAvailable, maintenanceEndTime ) => {
```

Comment:
As noted above, let's default `maintenanceEndTime = 0` :pray: (or perhaps `null`, since that's the starting value?)

Reply:
💯